cmd/geth/chaincmd.go

```diff
@@ -277,7 +277,7 @@ func createPorts(ipStr string, port int, size int) []int {
 // Create config for node i in the cluster
 func createNodeConfig(baseConfig gethConfig, enodes []*enode.Node, ip string, port int, size int, i int) gethConfig {
 	baseConfig.Node.HTTPHost = ip
-	baseConfig.Node.P2P.ListenAddr = fmt.Sprintf(":%d", port+i)
+	baseConfig.Node.P2P.ListenAddr = fmt.Sprintf(":%d", port)
```

Comment:
What was the reason for this change? The nodes in the cluster need to have different listen addresses.

Reply:
ListenAddr is not the same with enodes now

Reply:
if they are in one machine , they should listen different ports
if they are in different machines , they can listen the same ports ( or not)

but the listen port ,should keep the same with port in enodes

Reply:
for example
using node-deploy
<img width="1267" alt="image" src="https://github.com/bnb-chain/bsc/assets/122502194/3a02b9a9-c96c-4cb9-ac18-de2e6ab681ee">